### PR TITLE
Update Urbit to v1.18

### PR DESCRIPTION
**Pull request by yapishu**
- Updated Urbit to official v1.18 binary: https://github.com/urbit/vere/releases

### CODE REVIEW

File: urbit/umbrel-app.yml [hunk 1]
```diff
@@ -35,24 +35,14 @@ torOnly: false
...
-  -   Updated versioning scheme for pre-release binaries.
-  
-  
-  -   The code for the Vere runtime has been moved out of the urbit/urbit
-      repository, and now lives in urbit/vere.
-
-  Full release notes here: https://github.com/urbit/vere/releases/tag/vere-v1.17
+  Adds runtime support for the doccords system, which is documentation
+  parsed from comments in source code (Hoon’s version of docstrings).
+  Shoutout to ~datnut-pollen (@drbeefsupreme) for doing the huge amount of
```
Comment (nevets963) [inline urbit/umbrel-app.yml:40]: Due to usage of `>-` with this section. An empty line is needed here to ensure this ends up on a new line and not appended to the previous line.

I would also suggest a double newline to get empty space between sections, eg where you have the lines `This release accompanies` and `Full release note`.

This is a screenshot of how these notes appear atm:
Screenshot 2023-02-02 at 01 34 26

**Review by nevets963 — COMMENTED** [review]
@yapishu Thanks for consistently keeping Urbit up to date within Umbrel!

I only have some feedback re the formatting of the release notes.

Otherwise tested on `arm64` and `amd64`, and ready to merge after a small adjustment to the release notes 🚀

File: urbit/umbrel-app.yml (hunk 1 above)
Comment (yapishu) [reply urbit/umbrel-app.yml]: No problem, let me know if this looks better

File: urbit/umbrel-app.yml [hunk 2]
```diff
@@ -35,24 +35,19 @@ torOnly: false
...
-          
-  -   Updated versioning scheme for pre-release binaries.
-  
-  
-  -   The code for the Vere runtime has been moved out of the urbit/urbit
-      repository, and now lives in urbit/vere.
-
-  Full release notes here: https://github.com/urbit/vere/releases/tag/vere-v1.17
+
+  Adds runtime support for the doccords system, which is documentation
```
Comment (nevets963) [inline urbit/umbrel-app.yml:39]: Looks much better, thank you!

Screenshot 2023-02-02 at 02 15 35

Can you remove the empty line above this? After that, we're good to go.

Comment (yapishu) [reply urbit/umbrel-app.yml:38]: Gotcha, I misunderstood what you meant in the first comment. Removed

**Review by nevets963 — APPROVED** [review]
@yapishu Looks perfect!

Merging now 🚀

**Pull request merged**